train/training.py

```diff
@@ -93,6 +94,61 @@ def convert_labels_to_csv(data, tagging_output_file_path):
         print("An error occurred attempting to write to file at {0}:\n\n{1}".format(tagging_output_file_path,e))
         raise
 
+def upload_data_post_training(training_id):
+    tagging_output_file_path =  os.path.expanduser(config.get('tagged_output'))
+    function_url = config.get("url") + "/api/classification"
+    query = {
+        "userName": config.get('tagging_user')
+    }
+
+    # First, we need to get a mapping of class names to class ids
+    classes = ""
+    with open(tagging_output_file_path + "/tagged.csv") as f:
+        content = f.readlines()
+        for line in content:
+            class_name = line.strip().split(',')[1]
+            if class_name not in classes:
+                classes = classes + class_name + ","
+    
+    query["className"] = classes[:-1]
+    response = requests.get(function_url, params=query)
+    classification_name_to_class_id = response.json()
+    
+    # Now that we have a mapping, we create prediction labels in db
+    query = {
+        "userName": config.get('tagging_user'),
+        "trainingId": training_id
+    }
+    function_url = config.get("url") + "/api/labels"
+    payload_json = process_post_training_csv(tagging_output_file_path + "/tagged.csv", training_id, classification_name_to_class_id)
```

Comment:
@samiyaakhtar the post training file (prediction) is actually not this file. Let's have a method parameter called prediction_output_file or something. 

``` 
def upload_data_post_training(training_id, prediction_output_file):
...
```

If possible it's pass all arguments in the method signature. That way it becomes a static method and easier to test

Reply:
Done